tools/sense_studio/sense_studio.py

```diff
@@ -268,12 +268,15 @@ def add_header(r):
     return r
 
 
+# This context processor will inject a method into templates,
+# Which can be invoked like an ordinary method in HTML templates.
+# E.g. check navigation.html: line 1-2
```

Comment:
This is exactly what a docstring should be used for, i.e. triple quotes below the function signature